### DISCUSSION TITLE
Error: Color Mode switching darkmode to lightmode

### DISCUSSION THREAD

**Discussion by kimpoy31** [chakra-ui/chakra-ui#8183] [Q&A]
### I am getting this error on my console on development but my code works fine.
 app-index.js:32  Warning: Prop `className` did not match. Server: "__className_e66fe9 chakra-ui-light" Client: "__className_e66fe9"

![image](https://github.com/chakra-ui/chakra-ui/assets/150112883/244d4e31-a2ae-4dfe-b6e7-92e974bf729b)
![image](https://github.com/chakra-ui/chakra-ui/assets/150112883/36150d4f-fa0a-43a4-88b9-cf5bcdbb9a8e)
![image](https://github.com/chakra-ui/chakra-ui/assets/150112883/61dc5389-5b4b-46d4-90e9-1f1404618113)
![image](https://github.com/chakra-ui/chakra-ui/assets/150112883/5e425f70-e0f0-46dd-a727-da5abf300999)

**Comment by chyde** [comment]
I get this same error, and if you initialized theme to "dark", it will switch to light mode after initial render and hitting this error

**Reply by kimpoy31** [reply]
have you solved it? I am still not able to fix it.

**Comment by kimpoy31** [comment]
Add colorModeManager (Optional, for SSR)

Used this from chakraUi documentation.
